cmd/curio/run.go

```diff
@@ -135,7 +138,13 @@ var runCmd = &cli.Command{
 			return xerrors.Errorf("starting market RPCs: %w", err)
 		}
 
-		err = rpc.ListenAndServe(ctx, dependencies, shutdownChan) // Monitor for shutdown.
+		api, closer, err := lcli.GetFullNodeAPI(cctx)
```

Comment:
I though I already clarified that we don't need this in run in the original issue